packages/components/src/index.js

```diff
@@ -131,6 +132,7 @@ export {
 export { default as TreeSelect } from './tree-select';
 export { default as __experimentalTruncate } from './truncate';
 export { default as __experimentalUnitControl } from './unit-control';
+export { default as __experimentalView } from './view';
```

Comment:
I don't think we should export `View` unless we're ready to accept exposing the "guts" of G2. Are we sure we're ready for that?

Reply:
I think so. View would come in handy as we start building out UI like the [Typography Panel](https://github.com/WordPress/gutenberg/issues/27331).

Reply:
Yes, as long as we keep the experimental prefix while we ensure the APIs are stable, exporting should be fine.